### DISCUSSION TITLE
How do I add an outline to text layer

### DISCUSSION THREAD

**Discussion by Alkondr223322** [etro-js/etro#234] [Q&A]
Hi, I'm looking for a way to add an outline to the text in the Movie to make it more visible

`    const textLayer = new etro.layer.Text({ 

      startTime: startTime, 
      
      duration: durationStep, 
      
      text: sentence, 
      
      // x: 0, // default: 0 
      
      // y: 0, // default: 0 
      
      // width: 400, // default: null (full width) 
      
      // height: 400, // default: null (full height) 
      
      opacity: 1, // default: 1 
      
      //color: etro.parseColor('pink'), // default: new etro.Color(0, 0, 0, 1) randomColor({format: "rgba", alpha: 1}); 
      
      color: randomColor({format: "rgba", alpha: 1}), 
      
      font: '50px sans-serif', // default: '10px sans-serif' 
      
      textX: video[1]/2, // default: 0 
      
      textY: video[2] * 0.75, // default: 0 
      
      textAlign: 'center', // default: 'left' 
      
      textBaseline: 'alphabetic', // default: 'alphabetic' 
      
      textDirection: 'ltr', // default: 'ltr' 
      
      border: { // default: null (no border) 
        color: etro.parseColor('black'), 
        thickness: 10,
         
      }, 
      
      //background: etro.parseColor('red'), 
      
    });`

I tried doing it by adding the border, but I don't see any changes. Am I setting the border wrong?

**Comment by clabe45** [comment]
Hi, the text layer currently doesn't support adding a border. The [`color` option](https://etrojs.dev/docs/reference/layers/text#color) specifies the fill color for the text. If you want to create an issue for this contributors will be able to easily see it. I think it's a good idea to add a border.

**Reply by Alkondr223322** [reply]
Hello, thank you for the quick answer. Issue created, I'll just use a second text layer as a "shadow" for now
